client/webpack.config.js

```diff
@@ -180,9 +180,7 @@ const webpackConfig = {
 		chunkIds: isDevelopment || shouldEmitStats ? 'named' : 'natural',
 		minimize: shouldMinify,
 		minimizer: Minify( {
-			cache: process.env.CIRCLECI
-				? `${ process.env.HOME }/terser-cache/${ extraPath }`
-				: 'docker' !== process.env.CONTAINER,
```

Comment:
Did you intentionally change the docker exception here? We didn't put the cache into Docker because subsequent Docker images will not benefit from them.

Reply:
Yes I did, to treat terser cache like all other caches (eg: babel). These are deleted from the Docker image anyway, so it shouldn't affect the size:

```
RUN        WORKERS=$workers CALYPSO_ENV=production BUILD_TRANSLATION_CHUNKS=true yarn run build && rm -fr .cache
```

Nevertheless, I'll keep an eye on Docker image size to make sure it doesn't increase because this change